### ISSUE TITLE
Unable to Use!!

### ISSUE THREAD

**Issue by RamuSriram** [n0a/telegram-get-remote-ip#10]
It just shows "Capturing traffic, please wait..." after selecting interface and calling to a mutual contact in Telegram Desktop. Using Windows, downloaded all modules, what's the problem? Please help.

**Comment by n0a** [comment]
Tell me more information. What network interface are you use? Do you run script with admin priv?

**Comment by n0a** [comment]
And show me screenshots please if you have it.

**Comment by RamuSriram** [comment]
> And show me screenshots please if you have it.

Sure, here it is.
![IMG_20231019_003533_223](https://github.com/n0a/telegram-get-remote-ip/assets/71754546/bccb9ff6-7c27-430a-817a-a31ccc3e0943)

**Comment by RamuSriram** [comment]
> Tell me more information. What network interface are you use? Do you run script with admin priv?

Used Wifi network interface which I have connected to, and yes I tried in Command Prompt as Admin too.

**Comment by maskedyx** [comment]
Yeah I am getting the same issue here is the screenshot of the terminal
![image](https://github.com/n0a/telegram-get-remote-ip/assets/140275391/da7986e0-4131-4dd4-9943-76861e4558b1)

**Comment by n0a** [comment]
@RamuSriram @maskedyx 

Very strange. If you do not have any output in the terminal, then tshark does not capture packets. I'll see what can be done soon.

**Comment by Sputnik444** [comment]
Hello, this is due to the fact that the switch "Use peer-to-peer network for calls" is disabled in the call privacy settings, and in order for the IP address detection to work, this function must be enabled for both users. When this feature is disabled, calls pass through Telegram servers, which does not allow you to get an ip, when this feature is enabled, calls are redirected directly from one interlocutor to another interlocutor bypassing Telegram servers, so we can track the IP of the interlocutor.

**Comment by RamuSriram** [comment]
> Hello, this is due to the fact that the switch "Use peer-to-peer network for calls" is disabled in the call privacy settings, and in order for the IP address detection to work, this function must be enabled for both users. When this feature is disabled, calls pass through Telegram servers, which does not allow you to get an ip, when this feature is enabled, calls are redirected directly from one interlocutor to another interlocutor bypassing Telegram servers, so we can track the IP of the interlocutor.

Yeah, I know. And yeah peer-to-peer for calls is enabled already for both of them

**Comment by zhuxu** [comment]
> > Hello, this is due to the fact that the switch "Use peer-to-peer network for calls" is disabled in the call privacy settings, and in order for the IP address detection to work, this function must be enabled for both users. When this feature is disabled, calls pass through Telegram servers, which does not allow you to get an ip, when this feature is enabled, calls are redirected directly from one interlocutor to another interlocutor bypassing Telegram servers, so we can track the IP of the interlocutor.
> 
> Yeah, I know. And yeah peer-to-peer for calls is enabled already for both of them

you can test your network NAT type is what? while i got the same problem, my network NAT type is Symmetric NAT; i guess that is the problem when i check the packet flow by wireshark, while src client not direct connect to the dst client, instead, src and dst connected by use the telegram relay server

**Comment by anglee0323** [comment]
anyone fix that, the same issues
